<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextureFontCreatorGUIClass</class>
 <widget class="QMainWindow" name="TextureFontCreatorGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Texture Font Creator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="graphics.qrc">
    <normaloff>:/new/prefix1/Icon</normaloff>:/new/prefix1/Icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Font Input</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="3">
          <widget class="QPushButton" name="fontPathBrowseButton">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Render Settings</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Font</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="fontPathEdit"/>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="fontInputSizeSpinBox">
           <property name="suffix">
            <string>px</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>999.000000000000000</double>
           </property>
           <property name="value">
            <double>20.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Font Size</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="antialisedCheckBox">
           <property name="text">
            <string>Antialised</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="hintedCheckBox">
           <property name="text">
            <string>Hinted</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Font Output</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="powerOfTwoCheckBox">
           <property name="text">
            <string>Force power of two size image</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Font Info</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Font Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="fontNameLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Unknown</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Character Set Selection</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="ASCII_checkBox">
           <property name="text">
            <string>ASCII</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ISO_8859_15_checkBox">
           <property name="text">
            <string>ISO 8859-15 (Western Europe)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="japaneseHiraganaCheckbox">
           <property name="text">
            <string>Japanese (Hiragana)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="japaneseKatakanaCheckbox">
           <property name="text">
            <string>Japanese (Katakana)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="japaneseKanjiCheckbox">
           <property name="text">
            <string>Japanese (Jōyō Kanji)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="customCharacterSetGroupBox">
        <property name="title">
         <string>Custom Character Set</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QTextEdit" name="customCharacterSetTextEdit">
           <property name="acceptRichText">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="FontPreviewGroupBox">
        <property name="title">
         <string>Font Output Preview</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Zoom:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="zoomSlider">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>20</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGraphicsView" name="previewGraphicsView">
           <property name="cursor" stdset="0">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="dragMode">
            <enum>QGraphicsView::ScrollHandDrag</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="updatePreviewButton">
          <property name="text">
           <string>Update Preview</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_Texture_Font"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave_Texture_Font">
   <property name="text">
    <string>Save Texture Font</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="graphics.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>TextureFontCreatorGUIClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
